.polylintrc

```diff
@@ -1,5 +1,5 @@
 py:flake8 --exclude */migrations/*,*/settings/* %s
 py:isort -sl -rc -c %s
 scss:node_modules/.bin/stylelint --syntax scss %s
-html:node_modules/.bin/htmlhint -c .htmlhintrc %s
+html:node_modules/.bin/htmlhint -c .htmlhintrc -i **/email*.html %s
```

Comment:
As far as I can see, the only issue is the style tag in `email_base.html`. I would prefer not to exclude all email templates for that. Maybe just exclude `email_base.html`?